### PR TITLE
Changed min length on validator for user creation to 3 characters on use...

### PR DESCRIPTION
**Pull request by trideout**
Changed min length on validator for user creation to 3 characters on username and name

added unique verification on username and email fields
change suspended to default false in migrations
added username field to migrations

### CODE REVIEW

File: app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function up()
 		Schema::table('users', function(Blueprint $table)
 		{
 			$table->boolean('permissions')->default(1);
-			$table->boolean('suspended');
+			$table->boolean('suspended')->default(false);
```
Comment (cryptiklemur) [inline app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php]: Should be on two lines

File: app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ public function up()
 		Schema::table('users', function(Blueprint $table)
 		{
 			$table->boolean('permissions')->default(1);
```
Comment (cryptiklemur) [inline app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php]: Should be on two lines

File: .gitignore [hunk 3]
```diff
@@ -9,7 +9,6 @@ Thumbs.db
 /app/config/database.php
 app/config/database.php
```
Comment (cryptiklemur) [inline .gitignore:10]: duplicated? I know this wasnt you pfy, but mind fixing? Strip the forward slashes

File: app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php [hunk 4]
```diff
@@ -14,8 +14,10 @@ public function up()
 	{
 		Schema::table('users', function(Blueprint $table)
 		{
-			$table->boolean('permissions')->default(1);
-			$table->boolean('suspended');
+			$table->boolean('permissions')
+                ->default(1);
```
Comment (cryptiklemur) [inline app/database/migrations/2014_04_25_233547_add_permissions_suspended_to_users_table.php:18]: space these out further

**Comment by ericvanjohnson** [comment]
@aequasi I am going to test and merge then I will make those corrections.

**Pull request merged**